tests/test_others.py

```diff
@@ -220,3 +232,24 @@ def test_unk_error():
         assert str(e) == "'UNK' ligand name is not supported, please rename it, e.g. 'LIG'."
         return
     assert False
+
```

Comment:
Can we make the test towards unit testing?? Initialize the object SmilesConstraints, run the function, and then check if the resultant constraints are good to go.